### ISSUE TITLE
Lint/EndAlignment should support autocorrect

### ISSUE THREAD

**Issue by backus** [bbatsov/rubocop#3866]
Happy to help implement this. Expected this to be autocorrectable:

```sh
$ cat ex.rb
# frozen_string_literal: true
class Foo
  module Bar
end
end

$ rubocop -a ex.rb
Inspecting 1 file
W

Offenses:

ex.rb:3:3: C: Missing top-level module documentation comment.
  module Bar
  ^^^^^^
ex.rb:4:1: W: end at 4, 0 is not aligned with module at 3, 2.
end
^^^

1 file inspected, 2 offenses detected

$ cat ex.rb
# frozen_string_literal: true
class Foo
  module Bar
end
end
```

**Comment by backus** [comment]
Ah it has autocorrect but it is disabled by default. I didn't expect that...

**Comment by backus** [comment]
Looked into why. Explanation here https://github.com/bbatsov/rubocop/pull/1789#issuecomment-92308357
